src/material-examples/stepper-overview/stepper-overview-example.html

```diff
@@ -0,0 +1,23 @@
+<button (click)="isLinear=!isLinear" id="toggle-linear">Enable linear</button>
+<md-horizontal-stepper [linear]="isLinear">
+  <md-step [stepControl]="firstFormGroup">
+    <form [formGroup]="firstFormGroup">
+      <ng-template mdStepLabel>Step one</ng-template>
+      Content of step one
```

Comment:
Could you make the example a little more "real"? In general, users will get a better idea of how the component works when it more closely represents what they're be doing with it.

(these are the examples that show up on the docs site)